src/clr-addons/slider/slider.html

```diff
@@ -0,0 +1,56 @@
+<!--
+  ~ Copyright (c) 2018 Porsche Informatik. All Rights Reserved.
+  ~ This software is released under MIT license.
+  ~ The full license information can be found in LICENSE in the root directory of this project.
+  -->
+
+<div class="slider">
+  <h4>Value: {{_value}}</h4>
+  <div class="slider-action-field input" [hidden]="!_showsLabels">
```

Comment:
whats the reason for the input class here? the div is not an input, should have another class